helm/charts/determined/values.yaml

```diff
@@ -143,8 +142,42 @@ taskContainerDefaults:
   # (CPU notebooks, TensorBoards, zero-slot commands). If a pod spec is defined for an individual
   # task, that pod spec will replace the default one that is defined here. See
   # https://docs:determined.ai/latest/topic-guides/custom-pod-specs.html for more details.
-  # cpuPodSpec:
-  # gpuPodSpec:
+  cpuPodSpec:
+    apiVersion: v1
+    kind: Pod
+    metadata:
+      labels:
+        customLabel: cw-cpu-determined-pod
+    spec:
+      containers:
+        - name: determined-cpu-container
+          resources:
+            requests:
```

Comment:
This will only set pod resource requests on a CPU-only worker pod. Not GPU pods. You will need to add this to the gpuPodSpec too.